libs/components/src/lib/sidenav-disclosure/sidenav-disclosure.scss

```
@@ -0,0 +1,85 @@
+@use "../../../../shared/src/lib/sass/mixins/typography" as typography;
+@use "../../../../../node_modules/@vonage/vivid-tokens/dist/sass/themes/consts" as theme-consts;
+
+@use "../../../../shared/src/lib/sass/mixins/connotation/config" with (
+	$connotations: accent,
+	$shades: primary faint soft,
+	$default: accent
+);
+@use "../../../../shared/src/lib/sass/mixins/connotation" as connotation;
+
+@use "../../../../shared/src/lib/sass/mixins/appearance/config" as appearance-config with (
+	$appearances: ghost,
+	$states: idle hover active selected,
+	$default: ghost
+);
+@use "../../../../shared/src/lib/sass/mixins/appearance" as appearance;
+
+.control {
+	$min-block-size: 40px;
+	$gap: 8px;
+
+	@include typography.typography-cat-shorthand("body-2");
+	@include appearance.appearance;
+	@include connotation.connotation;
+
+	position: relative;
+	display: inline-flex;
+	box-sizing: border-box;
+	align-items: center;
+	background-color: var(#{appearance.get-appearance-token(fill)});
+	border-radius: 6px;
+	box-shadow: inset 0 0 0 1px var(#{appearance.get-appearance-token(outline)});
+	color: var(#{appearance.get-appearance-token(text)});
+	cursor: pointer;
+	gap: $gap;
+	hyphens: auto;
+	inline-size: 100%;
+	min-block-size: $min-block-size;
+	padding-block: 10px;
+	padding-inline: $gap;
+	text-decoration: none;
+	vertical-align: middle;
+	word-break: break-word;
+
+	@supports selector(:focus-visible) {
+		&:focus {
+			outline: none;
+		}
+	}
+
+	&.icon-only {
+		display: flex;
+		block-size: $min-block-size;
+		inline-size: $min-block-size;
+		place-content: center;
+	}
+
+	.toggleIcon {
+		position: absolute;
+		right: 16px;
```

why absolute? Its parent is flex- we can set it to be on the right like this:
```suggestion
		margin-inline-start: auto;
```

---

I don't see a chevron there...

---

@rachelbt Right, not the chevron but all your other comments